src/sagemaker/session.py

```diff
@@ -2819,7 +2823,17 @@ def create_model_package_from_containers(
             approval_status,
             description,
             drift_check_baselines=drift_check_baselines,
+            customer_metadata_properties=customer_metadata_properties,
         )
+        if model_package_group_name is not None:
+            try:
+                self.sagemaker_client.describe_model_package_group(
+                    ModelPackageGroupName=request["ModelPackageGroupName"]
+                )
+            except ClientError:
```

Comment:
Any way this exception can be made more specific? IE check the error code, or test for `sagemaker_client.exceptions.ResourceNotFound` if that API returns ResourceNotFound